src/helpers/isReply.js

```diff
@@ -10,8 +10,11 @@ const isReply = tweet => {
     tweet.in_reply_to_user_id ||
     tweet.in_reply_to_user_id_str ||
     tweet.in_reply_to_screen_name
-  )
+  ) {
     return true
+  }
+
+  return false

```

Comment:
does it not return false by default if it's not `true`?

Reply:
Kind of this will return undefined. https://repl.it/@EQuimper/isReply

The only place this function return is inside the if block. If you want it to return without saying false etc you can do this way https://repl.it/@EQuimper/isReplyV2

Also yes undefined is falsy but I think is better practice to do this or do the way in the isRelyV2